src/ExpressionEvaluator/CSharp/Test/ResultProvider/ExpansionTests.cs

```diff
@@ -69,6 +69,91 @@ public void Primitives()
             Verify(FormatResult("\"\"", CreateDkmClrValue("")), EvalResult("\"\"", "\"\"", "string", "\"\"", DkmEvaluationResultFlags.RawString, editableValue: "\"\""));
         }
 
+        [Fact, WorkItem(40179, "https://github.com/dotnet/roslyn/issues/40179")]
+        public void FieldMembersComeFirst()
+        {
+            var source =
+@"class C
+{
+    object Z0 { get => null; }
+    object m_z0 { get => null; }
+    object _z0 { get => null; }
+    object A0 { get => null; }
+    object _a0 { get => null; }
+    object _0 { get => null; }
+    object m_a0 { get => null; }
```

Comment:
> object m_a0 { get => null; } [](start = 3, length = 29)

LEt's change some of these to have getter and setter, and setter only to test all the combinations.

Reply:
Do we have a test for events with customer accessors?

---
In reply to: [357402633](https://github.com/dotnet/roslyn/pull/40210#discussion_r357402633) [](ancestors = 357402633)